lib/ansible/utils/collection_loader.py

```diff
@@ -38,17 +38,33 @@
 class AnsibleCollectionLoader(with_metaclass(Singleton, object)):
     def __init__(self, config=None):
         if config:
-            self._n_configured_paths = config.get_config_value('COLLECTIONS_PATHS')
+            paths = config.get_config_value('COLLECTIONS_PATHS')
         else:
-            self._n_configured_paths = os.environ.get('ANSIBLE_COLLECTIONS_PATHS', '').split(os.pathsep)
+            paths = os.environ.get('ANSIBLE_COLLECTIONS_PATHS', '').split(os.pathsep)
 
-        if isinstance(self._n_configured_paths, string_types):
-            self._n_configured_paths = [self._n_configured_paths]
-        elif self._n_configured_paths is None:
-            self._n_configured_paths = []
+        if isinstance(paths, string_types):
+            paths = [paths]
+        elif paths is None:
+            paths = []
 
         # expand any placeholders in configured paths
-        self._n_configured_paths = [to_native(os.path.expanduser(p), errors='surrogate_or_strict') for p in self._n_configured_paths]
+        paths = [
+            to_native(os.path.expanduser(p), errors='surrogate_or_strict')
+            for p in paths
+        ]
```

Comment:
This line doesn't need to be wrapped. It was under the column limit before, and it's even shorter now.

Reply:
I don't always use the line length as a metric to decide if I should go multiline. In this case, I wanted to separate nested expressions for easier consumption by the reader.
The fact that the linter has a huge line length setting doesn't mean that we should be trying to hit that limit.